src/components/SecretsModal/UniversalFields.js

```diff
@@ -34,31 +36,39 @@ const UniversalFields = props => {
         placeholder="secret-name"
         value={name}
         labelText="Name:"
-        onChange={handleChange}
+        onChange={handleChangeTextInput}
         invalid={invalidFields.indexOf('name') > -1}
         invalidText="Must be less than 563 characters, contain only lowercase alphanumeric character, . or -"
         autoComplete="off"
       />
       <NamespacesDropdown
         id="namespace"
-        selectedItem={{ id: selectedNamespace, text: selectedNamespace }}
-        onChange={handleChange}
+        selectedItem={
+          selectedNamespace
+            ? {
+                id: selectedNamespace,
+                text: selectedNamespace
+              }
+            : ''
+        }
+        onChange={handleChangeNamespace}
+        invalid={invalidFields.indexOf('namespace') > -1}
       />
       <Dropdown
         id="accessTo"
         titleText="Access To:"
+        helperText="These secrets are only for resources that are tekton-labelled, thus plain kube secrets required by other resource types (e.g. cluster, pullRequest, storage) are not covered by this UI."
```

Comment:
This is too much content for the `helperText`. When I said we should document this I meant in the README or similar, I should have made that clearer.